app/models/spree/related_products_configuration.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Spree
```

Comment:
There's a [Config class](https://github.com/solidusio-contrib/solidus_related_products/blob/c9c6ee5a025d8a3085220f3ba5690f0ef2edd3f6/lib/solidus_related_products/configuration.rb) now in master, taken from the extension template [[1]](https://github.com/solidusio/solidus_dev_support/blob/master/lib/solidus_dev_support/templates/extension/lib/%25file_name%25/configuration.rb.tt) [[2]](https://github.com/solidusio/solidus_dev_support/blob/858d0edf16a624bc02559c29d4b4240d9b7203f4/lib/solidus_dev_support/templates/extension/lib/generators/%25file_name%25/install/templates/initializer.rb.tt). I think it's important to use that one to be consistent across all extensions. Can we please move the configuration code to this pattern?